Modules/System/Azure Blob Services API/src/Helper/ABSOptionalParameters.Codeunit.al

```diff
@@ -334,14 +334,24 @@ codeunit 9047 "ABS Optional Parameters"
     end;
 
     /// <summary>
-    /// Specifies the maximum number of blobs to return
+    /// Specifies the maximum number of blobs to be fetch, from 1 to 5000 if not defined it will fetch 5000
+    /// If more than 5000 blobs then use NextMarker to fetch the next set of blobs
```

Comment:
Maybe it's worth adding the full details of the documentation to this parameter:

https://learn.microsoft.com/en-us/rest/api/storageservices/list-blobs?tabs=azure-ad#uri-parameters
> In certain cases, the service might return fewer results than specified by maxresults, and also return a continuation token.

> Setting maxresults to a value less than or equal to zero results in error response code 400 (Bad Request).

Reply:
Is now updated